recipes/pescador/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set version = "1.0.0" %}
+{% set hash_type = "md5" %}
+{% set hash_val = "5f3663fe66ce18f3f06be9ba5cbfbe1a" %}
+
+package:
+  name: pescador
+  version: {{ version }}
+
+source:
+  fn: pescador-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pescador/pescador-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  number: 1
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+  # noarch_python: True
+  # preserve_egg_dir: True
+  # entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - pescador = pescador:main
+    #
+    # Would create an entry point called pescador that calls pescador.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - joblib >=0.9
+    - six
+    - pyzmq
+    - numpy x.x
+
+  run:
+    - python
+    - joblib >=0.9
+    - six
+    - pyzmq
+    - numpy x.x
+
+test:
+  # Python imports
+  imports:
+    - pescador
+
+  # commands:
+    # You can put test commands to be run here.  Use this to test that the
+    # entry points work.
+
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  home: http://github.com/pescadores/pescador
+  license: ISC
+  summary: 'Multiplex generators for incremental learning'
```

Comment:
Thanks for the CR.  I'll handle it in the feedstock, if it's all the same to you. :grin: